core/orderbook.py

```
@@ -1,16 +1,15 @@
-# core/orderbook.py
+from collections import deque
```

Ruff (0.11.9)

`collections.deque` imported but unused

Remove unused import: `collections.deque`

(F401)